extensions/amp-addthis/validator-amp-addthis.protoascii

```diff
@@ -29,20 +29,95 @@ tags: {  # <amp-addthis>
   html_format: AMP
   tag_name: "AMP-ADDTHIS"
   requires_extension: "amp-addthis"
+  attrs: {
+    name: "data-attr-backgroundColor"
+  }
+  attrs: {
+    name: "data-attr-borderRadius"
+  }
+  attrs: {
+    name: "data-attr-counterColor"
+  }
+  attrs: {
+    name: "data-attr-counts"
+  }
+  attrs: {
+    name: "data-attr-countsFontSize"
+  }
+  attrs: {
+    name: "data-attr-desktopPosition"
+  }
+  attrs: {
+    name: "data-attr-elements"
+  }
+  attrs: {
+    name: "data-attr-hideDevice"
+  }
+  attrs: {
+    name: "data-attr-hideEmailSharingConfirmation"
+  }
+  attrs: {
+    name: "data-attr-hideLabel"
+  }
+  attrs: {
+    name: "data-attr-iconColor"
+  }
+  attrs: {
+    name: "data-attr-mobilePosition"
+  }
+  attrs: {
+    name: "data-attr-numPreferredServices"
+  }
+  attrs: {
+    name: "data-attr-offset"
+  }
+  attrs: {
+    name: "data-attr-originalServices"
+  }
+  attrs: {
+    name: "data-attr-postShareFollowMsg"
+  }
+  attrs: {
+    name: "data-attr-postShareRecommendedMsg"
+  }
+  attrs: {
+    name: "data-attr-postShareTitle"
+  }
+  attrs: {
+    name: "data-attr-responsive"
+  }
+  attrs: {
+    name: "data-attr-shareCountThreshold"
+  }
+  attrs: {
+    name: "data-attr-size"
+  }
+  attrs: {
+    name: "data-attr-style"
+  }
+  attrs: {
+    name: "data-attr-textColor"
+  }
+  attrs: {
+    name: "data-attr-thankyou"
+  }
+  attrs: {
+    name: "data-attr-titleFontSize"
+  }
+  attrs: {
+    name: "data-attr-__hideOnHomepage"
+  }
   attrs: {
     name: "data-class-name"
     mandatory: false
   }
   attrs: {
     name: "data-product-code"
-    trigger: {
-      also_requires_attr: "data-pub-id"
-    }
     mandatory_oneof: "['data-product-code', 'data-widget-id']"
   }
   attrs: {
     name: "data-pub-id"
-    mandatory: true
+    mandatory: false
```

Comment:
with this becoming false it is no longer needed to be listed as `data-pub-id` will always be allowed, just not required.

Reply:
i assume the same is true for `data-class-nane` also and I'm removing it too

Reply:
correct.